src/Generator.Rewrite/Extensions/IMethodSignatureExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Linq;
+using Mono.Cecil;
+
+namespace OpenTK.Rewrite.Extensions
+{
+    /// <summary>
+    /// Provides convenience extension methods for <see cref="IMethodSignature"/>
+    /// instances that are used in the OpenTK.Rewrite namespace.
+    /// </summary>
+    public static class IMethodSignatureExtensions
```

Comment:
Remove the I before the class name.

Reply:
Why? The `ICustomAttributeProviderExtensions` class also has it due to the extension methods being for an interface.

Reply:
Oh, I'm sorry! Ignore this.